CMakeLists.txt

```diff
@@ -1288,8 +1275,8 @@ else()
     target_link_libraries(${CSOUNDLIB} PRIVATE ${libcsound_LIBS})
 endif()
 
-if(USE_VCPKG)
-    target_link_libraries(${CSOUNDLIB} PRIVATE SndFile::sndfile)
+if(USE_LIBSNDFILE)
+    target_link_libraries(${CSOUNDLIB} PUBLIC SndFile::sndfile)
```

Comment:
Why is this moving from private to public? Are we exposing libsndfile headers via the API? Otherwise this should remain private. I have noticed weird issue with this library and linking libcsound though so it would be good to clear this up.